imports/ui/QuestionDisplay.jsx

```diff
@@ -425,7 +425,9 @@ export class QuestionDisplay extends Component {
           </div>
           : ''
         }
-
+        <div className='ql-question-solution'>
```

Comment:
The solution needs to only be displayed if the correct answer is being displayed. This happens when:
- In class, the prof chooses 'show correct' (for a student)
- During review, if a student clicks 'show correct' (for a student)
- For the prof always, except when the prof is sharing their screen (I think that is "desktop mode" - see how displaying the correct checkmark is handled in, I think, session.jsx)

Except for the last item, you just need to replicate the conditional statements that decide whether to show correct in QuestionDisplay. 

Finally, there should be a check to show the solution only if it is non-empty. If empty, either display nothing, or display "solution not provided".